src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/entities/Study.java

```diff
@@ -61,7 +62,13 @@
     @Column(nullable = false)
     private String center;
 
-    @ApiModelProperty(position = 5, dataType = "java.lang.String", notes = "Url to a Taxonomy")
+    @ApiModelProperty(position = 5, required = true)
```

Comment:
please provide the example/format "yyyy-mm-dd" so that user knows the format while post.

Reply:
Added